GitCommands/Git/GitModule.cs

```diff
@@ -130,26 +152,35 @@ static bool HasGitModulesFile(string path)
             }
         }
 
-        /// <summary>
-        /// Gets the directory which contains the git repository.
-        /// </summary>
+        /// <inherit/>
         public string WorkingDir { get; }
 
         /// <summary>
-        /// Gets the access to the current git executable associated with this module.
+        /// GitVersion for the default GitExecutable.
         /// </summary>
+        public GitVersion GitVersion => GitVersion.CurrentVersion(_wslDistro);
+
+        /// <inherit/>
         public IExecutable GitExecutable => _gitExecutable;
 
-        /// <summary>
-        /// Gets the access to the current git executable associated with this module.
-        /// </summary>
+        /// <inherit/>
         public IGitCommandRunner GitCommandRunner => _gitCommandRunner;
 
         /// <summary>
         /// Gets the location of .git directory for the current working folder.
         /// </summary>
         public string WorkingDirGitDir { get; private set; }
 
+        /// <inherit/>
+        public string GetPathForGitExec(string? path) => PathUtil.GetRepoPath(path, _wslDistro);
+
+        /// <summary>
+        /// Convert a path to native (Windows) format.
+        /// </summary>
+        /// <param name="path">Path as seen by the Git executable, possibly WSL Git.</param>
+        /// <returns>Path in in Windows format with native file separators.</returns>
+        public string GetPathForNative(string path) => PathUtil.GetNativePath(path, _wslDistro);
```

Comment:
This must be added to `IGitModule` as well.

Reply:
I had planned to not add this to the interface, changing.